docs/specs/editorconfig-in-the-compiler.md

```diff
@@ -0,0 +1,111 @@
+All-capital words are interpreted per [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+# Compiler Command Arguments
+
+The compiler *MUST* receive a new switch, `/editorconfig`, which accepts a single
+file which is an .editorconfig file that may apply to one or more files in the
+compilation. Multiple files are passed with multiple `/editorconfig` switches,
+the same way you can pass multiple references with /reference or multiple analyzers
+with /analyzer. There is no short form of the argument. Files passed with relative
```

Comment:
@tannergooding couple of reasons:

1. From a compiler perspective this too random.  It means the output of my build, in particular the success or failure, is dependent upon where on the machine my code is checked out to.  Code may build fine on my machine but fail on yours because last week you copied an .editorconfig to your home directory.
1. Doesn't meet the core IDE scenarios which don't strictly align to editorconfig rules.